build/Dockerfile

```diff
@@ -1,4 +1,14 @@
-FROM golang:1.13.7-alpine3.11 as builder
+FROM registry.access.redhat.com/devtools/go-toolset-rhel7:1.12.12-4 as builder
+
+ENV PATH=/opt/rh/go-toolset-1.12/root/usr/bin:$PATH \
+    GOPATH=/go/
+
+USER root
+
+RUN case $(uname -m) in \
```

Comment:
I'm not sure what it does, just copied/pasted from che-operator Dockerfile. Sorry about that =)

Reply:
right now it does nothing but set a variable `ARCH` to one of two values based on the architecture of the platform.

You probably want to then use this when running the go compilation on line 75...

```
- RUN CGO_ENABLED=0 GOOS=linux ...
+ RUN CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} ...
```

Also if you want to support ppc64le as well as x86_64/amd64 and s390x, you could replace the case statement with this:

```
ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH="amd64"; fi
```